client/pullFromServer.go

```diff
@@ -1,17 +1,25 @@
 package main
 
 import (
-	"encoding/binary"
 	"fmt"
 	"net"
 	"os"
 	"path/filepath"
-	"strconv"
 	"strings"
 )
 
 func pullFromServer(conn net.Conn) {
```

Comment:
De requestString is enkel voor testen neem ik aan? In de toekomst wordt deze uit het request gehaald toch?

Reply:
Ja, als het goed is wel. Ik had de create header en create body functie naar 1 functie gezet ipv dat die overal was. Alleen goed toegepast op authorization maar nog niet naar de rest gekeken omdat dat wss toch aangepast moest worden.